smart-contracts/osmosis/contracts/cl-vault/src/vault/deposit.rs

```diff
@@ -118,33 +93,42 @@ pub(crate) fn execute_any_deposit(
             (deposit_info.base_deposit, deposit_info.quote_deposit),
         ),
     )?;
-    let swap_calc_result = calculate_swap_amount(
-        deps,
+
+    // Get TWAP price
```

Comment:
that comment is redundant. The function is already called get_twap_price.

Reply:
yep indeed, sometimes i put TODO comments during the development and then i just remove the TODO leaving the useless comment there. I will remove that